.gitattributes

```diff
@@ -0,0 +1,19 @@
+* text eol=lf
```

Comment:
@ChrisMcKee I'm not sure I fully get why this is needed. It's not in any of our other modules, so I'd like to chat through this a bit before approving. Can you explain this a bit more?

Reply:
https://www.git-scm.com/docs/gitattributes 
Its not really a problem that needs fixing so long as no one ever uses windows to edit these projects. 
If you're not on windows the line endings will vary depending on the settings on the machine. I work heavily in .net so subsequently git will try to crlf the line endings. 

Its clean git practice like setting spacing standards etc with editorconfigs and the like. Makes it easier to automate and enforce how your repos maintained 😄

Reply:
Could cut this file down of course to just markdown and tf; I lazily copied it from my geodesic

Reply:
removed all but the wildcard